v2/bundle/jwtbundle/set.go

```diff
@@ -0,0 +1,36 @@
+package jwtbundle
+
+import (
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// Set is a set of bundles, keyed by trust domain.
+type Set struct{}
+
+// NewSet creates a new, empty set.
+func NewSet(bundles ...*Bundle) *Set {
+	panic("not implemented")
+}
+
+// Insert inserts a new bundle into the set. If a bundle already exists for the
+// trust domain, the existing bundle is replaced.
+func (s *Set) Insert(bundle *Bundle) {
+	panic("not implemented")
+}
+
+// Remove removes the bundle for the given trust domain. True is returned if
+// the bundle is present (and removed), otherwise false.
+func (s *Set) Remove(trustDomain spiffeid.TrustDomain) bool {
```

Comment:
@evan2645 and I chatted IRL and decided `Remove` didn't need to return a bool. If a caller wants to know if there was something to remove, they can call `Has` first...